app/views/users/_profile.html.erb

```diff
@@ -4,7 +4,7 @@
 
 <div class="crayons-card mb-6 p-6 grid gap-6">
   <div class="flex items-center">
-    <a href="https://<%= ApplicationConfig["APP_DOMAIN"] %>/<%= current_user.username %>" aria-label="Go to your profile page" class="mr-2">
+    <a href="<%= app_url(current_user.username) %>" aria-label="Go to your profile page" class="mr-2">
```

Comment:
this can also be `user_url(current_user)` though the result is the same for now :D